click_odoo_contrib/initaddonhashes.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# Copyright 2024 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+import click
+import click_odoo
+
+from .update import _get_ignore_addons, _save_installed_checksums
```

Comment:
If we do this I'd rather have a separate refactoring commit to move these functions in a separate python file.

Or else, add a `--only-compute-hashes` to `click-odoo-update` ?

Reply:
Yes i would do that it is simpler and fits.